provision/infinispan/ispn-helm/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Default values for ispn-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 3
```

Comment:
In the bash scripts this is 2.

Reply:
I don't have a strong opinion if it should be 2 or 3 :shrug: